generate/templates/partials/async_function.cc

```diff
@@ -10,6 +10,11 @@ NAN_METHOD({{ cppClassName }}::{{ cppFunctionName }}) {
 
   baton->error_code = GIT_OK;
   baton->error = NULL;
+  {%if cppClassName == "GitStatus" %}
```

Comment:
This seems too specific. Check shouldAlloc flag. It should be able to generate this stub without adding this special condition.